src/renderer/components/screens/AccountsScreen.tsx

```diff
@@ -142,6 +150,17 @@ export default function AccountsScreen({
   const tx = useTranslationFunction()
   const [logins, setLogins] = useState(null)
 
+  const [sync_all_accounts, set_sync_all_accounts] = useState(null)
```

Comment:
When did we switch to snake_case for variables? I thought we enforced camelCase somewhere?